### ISSUE TITLE
clay has poor breach handling

### ISSUE THREAD

**Issue by belisarius222** [urbit/urbit#5968]
Old-style Ames ducts (without rift in them) associated with peers that have since breached accumulated for a while in Clay on publisher ships.  This creates a small, but bounded space leak. It also causes affected ships to send Ames packets to other ships that the other ship crashes on with an ames `dangling bone` error.

These ames errors are benign.  We should clean this up at some point, but the severity of this issue is bounded and not severe.

**Comment by joemfb** [comment]
see #5852

**Comment by zalberico** [comment]
@belisarius222 it's not clear to me from the summary what the actual task is (what needs to be cleaned up). Sounds like clay needs to clean up old ames ducts of ships that have breached? Is it possible to be more clear about what someone would have to do to implement this?

Basically goal for me would be "clay has poor breach handling" -> "Clay should clear old-style ames ducts of breached ships" with maybe some basic idea of how this should be done. Might be a good issue for a new hire fix? Probably low priority given the number of affected ships are only historical ones before the gall request queue fix if I understand correctly.

**Comment by belisarius222** [comment]
Yes, someone would have to write code to identify and clear old-style Ames ducts to breached ships.  It's low priority because this was a small leak that was not that common and is no longer increasing in size, so it's just not high on our list of problems.

It would be a small piece of code, but maybe not that easy to write, since it involves detailed understanding of the duct system
